sdk/documentintelligence/azure-ai-documentintelligence/azure/ai/documentintelligence/__init__.py

```diff
@@ -5,21 +5,30 @@
 # Code generated by Microsoft (R) Python Code Generator.
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
+# pylint: disable=wrong-import-position
 
-from ._patch import DocumentIntelligenceClient
-from ._patch import DocumentIntelligenceAdministrationClient
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from ._patch import *  # pylint: disable=unused-wildcard-import
+
+from ._client import DocumentIntelligenceClient  # type: ignore
```

Comment:
are these two type ignores needed?

Reply:
These are coming from the typespec generator. I can open an issue asking why they added this